core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -27,7 +27,9 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   appType: AppType,
                                   diskConfig: Option[PersistentDiskRequest],
                                   labels: LabelMap = Map.empty,
-                                  customEnvironmentVariables: Map[String, String])
+                                  customEnvironmentVariables: Map[String, String],
+                                  descriptorPath: Option[String],
+                                  extraArgs: List[String])
```

Comment:
what does `extraArgs` will look like?

Reply:
It's kind of like https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/

For example:
```
    command: ["printenv"]
    args: ["HOSTNAME", "KUBERNETES_PORT"]
```

Basically the app.yaml defines a `command` and `args` to startup the app. `extraArgs` exists in case the app accepts extra dynamic arguments that don't necessarily belong in the app descriptor (a path to a data file, for example) but the user might want to specify at app launch time.

 I did this originally mostly for `cirrocumulus` which requires a data path at app launch time.